cmd/node/config/prefs.toml

```diff
@@ -50,10 +50,28 @@
    #    { File = "external.toml", Path = "ElasticSearchConnector.Enabled", Value = "true" }
    #]
 
+# BlockProcessingCutoff can be used to stop processing blocks at a certain round, nonce or epoch.
+# This can be useful for snapshotting different stuff and also for debugging purposes.
+[BlockProcessingCutoff]
+   # If set to true, the node will stop at the given coordinate
+   Enabled = false
+
+   # Mode represents the cutoff mode. possible values: "pause" or "process-error".
+   # "pause" mode will halt the processing at the block with the given coordinates. Useful for snapshots/analytics
+   # "process-error" will return an error when processing the block with the given coordinates. Useful for debugging
+   Mode = "pause"
+
+   # CutoffTrigger represents the kind of coordinate to look after when cutting off the processing.
+   # Possible values: "round", "nonce", or "epoch"
+   CutoffTrigger = "round"
+
+   # The minimum value of the cutoff. For example, if CutoffType is set to "round", and Value to 20, then the node will stop processing at round 20+
+   Value = 20
```

Comment:
we should default this to 0 or something like this.

Reply:
done